### DISCUSSION TITLE
v0.10.5: Unstable Release

### DISCUSSION THREAD

**Discussion by Amazinite** [endless-sky/endless-sky#9728] [Announcements]
This is an unstable release, containing big changes that may introduce new bugs.

This update turns its focus back to human space, bringing with it improvements and expansions of both piracy and mining gameplay, as well as continuing the trend of UI and UX improvements.

Some other major changes and additions are:
* Expanded the mechanics behind asteroid mining, including the addition of mining lasers that increase the yield of asteroids, tractor beams that pull ores toward your ship, dedicated mining ships in human space, and a new mission in the Sparrow introduction that goes over the basics of asteroid mining.
* Added a small pirate region to the south, home to the remnants of a collapsed government: the Confederated Councils of the Outer Rim. Includes the addition of various new pirate ships and weapons. This is the first of a number of planned expansions to pirate content and gameplay, with the goal of making it so that playing as a pirate is a more viable option. Stay tuned for more in future updates.
* You can now review past hails and status messages by opening the messages log panel, default "/".
* Lowered the brightness of the center of the galaxy sprite to make the system names in the galaxy's core more readable.
* Plugins can now customize the services that a spaceport offers, and even the name that appears on the spaceport button.
* The state of your capslock key is now tied to the state of fast-forward in the game if your capslock key is bound to your fast-forward control. This means that toggling your capslock key even when the game window is not in focus will also toggle the game's fast-forward.
* The list of plugins and the selected plugin description in the plugins panel can now scroll, instead of causing too many plugins or verbose descriptions to overflow the menu.

Bug fixes:
* Planetary security will now properly scan illegal goods in your fleet's cargo. Previously, only illegal installed outfits could be scanned.
* Improved the collision detection of flotsam, preventing cases where flying over a flotsam at high velocity could cause it to phase through your ship and not be collected.
* Various fixes to missions in the Coalition intro storylines.

In addition to all the changes to the game from this update, we also have some great news outside of the game, including the creation of an [official trailer](https://www.youtube.com/watch?v=Nf3TEeSCkcM) and a new publishing location on [GOG](https://www.gog.com/en/game/endless_sky)!

For a complete list of changes, as well as further detail about the summarized changes, see the [changelog](https://github.com/endless-sky/endless-sky/blob/master/changelog). Special thanks to those who contributed to this release: @1010todd, @alexrovw, @Amazinite, @Anarchist2, @AntimatterReactor, @Arachi-Lover, @Azure3141, @beccabunny, @bene-dictator, @Corraban2, @Daeridanii1, @danaris, @DarcyManoel, @Ebreus, @EjoThims, @Ember369, @flowers9, @Galaucus, @hexagonrecursion, @Hurleveur, @jarmokivekas, @kaol, @Koranir, @LazerLit, @LepRyot, @MCOfficer, @mOctave, @opusforlife2, @Quantumshark, @quyykk, @RedDeadUndead, @RisingLeaf, @roadrunner56, @Saugia, @Tangle10, @The-Legendary-M, @TheMarksman-ES, @thewierdnut, @ThrawnCA, @tibetiroka, @TomGoodIdea, @UnorderedSigh, @VeryGoodDog, @warp-core, @Wedge009, @williaji, @ziproot, and @Zitchas.

If you are using the high-dpi plugin, you should download and install the [updated version](https://github.com/endless-sky/endless-sky-high-dpi/releases/tag/v0.10.5) that goes with this release.

![image](https://github.com/endless-sky/endless-sky/assets/17688683/4e0c255d-edd7-4ac6-8307-7fde7e6f78ed)

<hr /><em>This discussion was created from the release <a href='https://github.com/endless-sky/endless-sky/releases/tag/v0.10.5'>v0.10.5: Unstable Release</a>.</em>

**Comment by brendanjones** [comment]
Amazing work, everyone involved! Absolutely loving that trailer, and I can’t wait to boot up a new pilot to try the new pirate content.
